Testscripts/Windows/FCOPY-REPEATED-DELETE.ps1

```diff
@@ -132,29 +137,39 @@ if (-not $sts) {
         }
         Write-LogInfo "File has been successfully copied to guest VM '${vmName}'"
 
-        $sts = Check-FileInLinuxGuest  -vmPassword $VMPassword -vmPort $VMPort -vmUserName $VMUserName -ipv4 $Ipv4   -fileName "/mnt/test/$testfile"
+        $sts = Check-FileInLinuxGuest  -vmPassword $VMPassword -vmPort $VMPort -vmUserName $VMUserName -ipv4 $Ipv4 -fileName "/mnt/test/$testfile" -checkSize $true
         if (-not $sts) {
             Write-LogErr "File check error on the guest VM '${vmName}'!"
             return "FAIL"
             break
         }
-        Write-LogInfo "The file copied matches the ${originalFileSize} size."
+        if (($sts/1gb) -eq ($fileSize/1gb) ) {
+            Write-LogInfo "The file copied matches the $fileSize size."
```

Comment:
as well, can you also do md5sum before/after file-copy?

Reply:
> as well, can you also do md5sum before/after file-copy?

In FCOPY-NON-ASCII.ps1 we do md5sum for the file.
It might go as a separate work item to maybe compute md5sum for all fcopy tests, but not in this PR.

Reply:
added to #184